src/Identity/Controllers/SsoController.cs

```diff
@@ -108,23 +109,23 @@ public async Task<IActionResult> Login(string returnUrl)
 
             return RedirectToAction(nameof(ExternalChallenge), new
             {
-                domain_hint = domainHint,
+                domainHint = domainHint,
```

Comment:
For future reference, like the `returnUrl`, etc. below this, you don't need the `=` assignment here as the named property of the anonymous object will simply take the variable name passed to it.